authors/apps/authentication/models.py

```diff
@@ -102,12 +102,12 @@ def __str__(self):
 
     @property
     def get_full_name(self):
-      """
-      This method is required by Django for things like handling emails.
-      Typically, this would be the user's first and last name. Since we do
-      not store the user's real name, we return their username instead.
-      """
-      return self.username
+        """
```

Comment:
What's with this change? Why do you need to add extra spaces here?

Reply:
I added extra spaces to fix pep8 errors to do with improper indentation